docs/CodeDevelopment/prerequisites.md

```diff
@@ -44,12 +44,23 @@ See component list here for more options. https://docs.microsoft.com/en-us/visua
     ``` cmd
     https://aka.ms/vs/16/release/vs_buildtools.exe
     ```
-2. Install from cmd line with required features (this set will change overtime).
+2. Install from cmd line with required features (this set will change over time).
     ``` cmd
     C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache --installPath C:\BuildTools --add Microsoft.VisualStudio.Component.VC.CoreBuildTools --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK.17763 --add Microsoft.VisualStudio.Component.VC.Tools.ARM --add Microsoft.VisualStudio.Component.VC.Tools.ARM64
     ```
 See component list here for more options. https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2019
 
+### Optional - Windows Driver Kit (required to sign firmware update Capsules)
```

Comment:
Since WDK is needed for Inf2Cat.exe lets update this to be accurate.

Reply:
Updated